mos_tests/neutron/python_tests/base.py

```diff
@@ -126,7 +126,7 @@ def execute():
 
     def check_ping_from_vm(self, vm, vm_keypair=None, ip_to_ping=None,
                            timeout=3 * 60, vm_login='cirros',
-                           vm_password='cubswin:)'):
+                           vm_password='cubswin:)', should_be_available=True):
```

Comment:
May be better make 3 methods:
1. run_ping_from_vm (which return exit code)
2. check_ping_from_vm (which call 1st and check exit code is 0)
3. check_no_ping_from_vm (which call 1st and check exit code is not 0)